client/me/help/help-courses/index.jsx

```diff
@@ -16,16 +16,52 @@ import {
 import { getCurrentUserId } from 'state/current-user/selectors';
 import { PLAN_BUSINESS } from 'lib/plans/constants';
 
+const courses = [
+	{
+		title: i18n.translate( 'How to Make a Business Site on WordPress.com' ),
```

Comment:
If the user switched languages this won't be re-translated since `translate` is only evaluated on module startup. However if this is just mock data and will be replaced with an API call I don't think it's a problem for now.